src/init.cpp

```diff
@@ -154,6 +154,7 @@ class CCoinsViewErrorCatcher : public CCoinsViewBacked
 
 static CCoinsViewDB *pcoinsdbview = NULL;
 static CCoinsViewErrorCatcher *pcoinscatcher = NULL;
+static boost::scoped_ptr<ECCVerifyHandle> verify_handle;
```

Comment:
bike-shedding: s/verify_handle/globalVerifyHandle or something of the short to make its scope clearer.